src/backends/caffe/caffelib.cc

```diff
@@ -4010,18 +3946,16 @@ void CaffeLib<TInputConnectorStrategy,TOutputConnectorStrategy,TMLModel>::update
                     this->_logger->warn("dice contour amplitude unrecognized, float expected");
                   }
               }
+			this->_logger->warn("dice contour is not working (2020 - 05 - 19) disabling it");
+			contours = caffe::DiceCoefLossParameter::NO;
+
```

Comment:
Let's fail the creation / job if contour is selected, no ?

Reply:
done